src/Backend/Modules/Pages/Actions/Add.php

```diff
@@ -121,6 +128,19 @@ private function loadForm(): void
         // image related fields
         $this->form->addImage('image');
 
+        // just execute if the site is multi-language
+        if ($this->getContainer()->getParameter('site.multilanguage')) {
```

Comment:
I should have to test it again but as I remember the ALT-tags for alle the languages are correctly set, because the (revision)-ID of the page in each language are the same. I will try to test it again tomorrow and give a confirmation.

Reply:
@carakas I have reinstalled, and all the pages are correctly linked with the good HREF lang tags. As far as I can see, it looks good :)

Reply:
nice, awesome work :)